core/deployment/src/main/java/io/quarkus/deployment/pkg/steps/JarResultBuildStep.java

```diff
@@ -393,8 +393,10 @@ private void buildUberJar0(CurateOutcomeBuildItem curateOutcomeBuildItem,
             for (Map.Entry<String, Set<AppDependency>> entry : duplicateCatcher.entrySet()) {
                 if (entry.getValue().size() > 1) {
                     if (explained.add(entry.getValue())) {
-                        log.warn("Dependencies with duplicate files detected. The dependencies " + entry.getValue()
-                                + " contain duplicate files, e.g. " + entry.getKey());
+                        if (!"module-info.class".endsWith(entry.getKey())) {
```

Comment:
I think it would probably be better to add it here and ignore it altogether: https://github.com/quarkusio/quarkus/blob/main/core/deployment/src/main/java/io/quarkus/deployment/pkg/steps/JarResultBuildStep.java#L105

WDYT?

Reply:
@gsmet, 

I didn't add it there because it would have to be dynamic and there seems to be too static.

e. g:
```
META-INF/versions/{param_dynamic}/module-info.class
META-INF/versions/9/module-info.class
META-INF/versions/11/module-info.class
```

Reply:
@gsmet, wdyt?
I couldn't think of anything better.

Reply:
@gsmet  - sorry to bother, but I guess you have turned it the other way around:

instead of 
```
if (!entry.getKey().endsWith("module-info.class"))
```
you have written:
```
if (!"module-info.class".endsWith(entry.getKey()))
```
which hardly ever would be the case, or?

Have found it still written the same way in the 2.6.1.Final